CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # salsify_rubocop
 
+## 0.85.0
+- Upgrate to `rubocop` v0.85.0
+- Enable enforcement of Gem comments when specifying versions or sources 
+- Drop support for Ruby 2.3, as rubocop v0.81+ doesn't support it anymore
```

Comment:
Should we add a `spec.required_ruby_version = '>= 2.3'` statement to the gemspec to enforce this?

Reply:
Done. What happened to the Travis checks though? 🤔

Reply:
Weird. I updated the branch protection rules to require a green Travis build before merging.